### PR TITLE
feat(mrs): cluster support prepaid

### PR DESCRIPTION
**Pull request by zhangtao505**
**What this PR does / why we need it**:
cluster support prepaid.
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format,
  will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:
cluster support prepaid.

```release-note
```

### CODE REVIEW

File: openstack/mrs/v2/clusters/requests.go [hunk 1]
```diff
@@ -162,7 +162,21 @@ type ChargeInfo struct {
...
+	//   year: indicates that the unit is year.
+	PeriodType string `json:"periodType,omitempty"`
+	// Specifies the subscription term. This parameter is valid and mandatory only when chargingMode is set to prePaid.
+	//   When periodType is set to month, the parameter value ranges from 1 to 9.
+	//   When periodType is set to year, the parameter value ranges from 1 to 3.
+	PeriodNum int `json:"periodNum,omitempty"`
+	// Specifies whether to pay immediately. This parameter is valid only when chargingMode is set to prePaid. The default value is false.
+	//   false: indicates not to pay immediately after an order is created.
+	//   true: indicates to pay immediately after an order is created. The system will automatically deduct fees from the account balance.
+	IsAutoPay string `json:"isAutoPay,omitempty"`
```
Comment (Lance52259) [inline openstack/mrs/v2/clusters/requests.go:179]:
```suggestion
	PeriodType string `json:"period_type,omitempty"`
	// Specifies the subscription term. This parameter is valid and mandatory only when chargingMode is set to prePaid.
	//   When periodType is set to month, the parameter value ranges from 1 to 9.
	//   When periodType is set to year, the parameter value ranges from 1 to 3.
	PeriodNum int `json:"period_num,omitempty"`
	// Specifies whether to pay immediately. This parameter is valid only when chargingMode is set to prePaid. The default value is false.
	//   false: indicates not to pay immediately after an order is created.
	//   true: indicates to pay immediately after an order is created. The system will automatically deduct fees from the account balance.
	IsAutoPay string `json:"is_auto_pay,omitempty"`
```

File: openstack/mrs/v2/clusters/requests.go [hunk 2]
```diff
@@ -448,3 +462,16 @@ func Create(client *golangsdk.ServiceClient, opts CreateOptsBuilder) (r CreateRe
...
+	b, err := opts.ToClusterCreateMap()
+	if err != nil {
+		r.Err = err
+		return
+	}
+
+	reqOpt := &golangsdk.RequestOpts{OkCodes: []int{200}}
+	_, r.Err = client.Post(rootURL(client), b, &r.Body, reqOpt)
+	return
+}
```
Comment (Lance52259) [inline openstack/mrs/v2/clusters/requests.go:477]: There is unnecessary to support a extra function.
Put the `OrderId` into the `Cluster` structure just okey.

Comment (zhangtao505) [reply openstack/mrs/v2/clusters/requests.go]: done

File: openstack/mrs/v2/clusters/requests.go (hunk 1 above)
Comment (zhangtao505) [reply openstack/mrs/v2/clusters/requests.go]: done

File: openstack/mrs/v2/clusters/requests.go [hunk 3]
```diff
@@ -162,7 +162,21 @@ type ChargeInfo struct {
...
+	//   year: indicates that the unit is year.
+	PeriodType string `json:"period_type,omitempty"`
+	// Specifies the subscription term. This parameter is valid and mandatory only when chargingMode is set to prePaid.
+	//   When periodType is set to month, the parameter value ranges from 1 to 9.
+	//   When periodType is set to year, the parameter value ranges from 1 to 3.
+	PeriodNum int `json:"period_num,omitempty"`
+	// Specifies whether to pay immediately. This parameter is valid only when chargingMode is set to prePaid. The default value is false.
+	//   false: indicates not to pay immediately after an order is created.
+	//   true: indicates to pay immediately after an order is created. The system will automatically deduct fees from the account balance.
+	IsAutoPay string `json:"is_auto_pay,omitempty"`
```
Comment (Lance52259) [inline openstack/mrs/v2/clusters/requests.go:179]:
```suggestion
	IsAutoPay bool `json:"is_auto_pay,omitempty"`
```

Comment (zhangtao505) [reply openstack/mrs/v2/clusters/requests.go]: done

**Pull request merged**